src/pyosmeta/parse_issues.py

```diff
@@ -175,106 +221,47 @@ def get_last_commit(self, repo: str) -> str:
-gh_stats = [
-    "name",
-    "description",
-    "homepage",
-    "created_at",
-    "stargazers_count",
-    "watchers_count",
-    "stargazers_count",
-    "forks",
-    "open_issues_count",
-    "forks_count",
-]
-
-stats_list = gh_stats
-url = "https://api.github.com/repos/earthlab/earthpy"
-
-# meta = issueProcess.get_repo_meta(url, stats_list)
-
-# Get gh metadata for each package submission
-all_repo_meta = {}
-for package_name in all_repos.keys():
-    print(package_name)
-    all_repo_meta[package_name] = issueProcess.get_repo_meta(repo, gh_stats)
-
-    all_repo_meta[package_name]["contrib_count"] = issueProcess.get_repo_contribs(repo)
-    all_repo_meta[package_name]["last_commit"] = issueProcess.get_last_commit(repo)
-    # Add github meta to review metadata
-    review[package_name]["gh_meta"] = all_repo_meta[package_name]
-
-
-filename = "packages.yml"
-with open(filename, "w") as file:
-    # Create YAML object with RoundTripDumper to keep key order intact
-    yaml = ruamel.yaml.YAML(typ="rt")
-    # Set the indent parameter to 2 for the yaml output
-    yaml.indent(mapping=4, sequence=4, offset=2)
-    yaml.dump(review, file)
 
-# https://api.github.com/repos/pyopensci/python-package-guide/commits
+        return self._clean_date(date)
+
+    # issue_body_list = body_data
 
+    def get_categories(self, issue_body_list: list) -> list:
+        """Parse through a pyos issue and grab the categories associated
+        with a package
 
-"""
-Add: to data
-  date-accepted: 2021-12-29
-  highlight:
-  docs-url: "https://jointly.readthedocs.io"
-  citation-link:
-
-
-# Create dict with info
-# Using create contrib file method from other script
-
-  - package-name: jointly
-  description: "Jointly: A Python package for synchronizing multiple sensors with accelerometer data"
-  maintainer:
-    [
-      "Arne Herdick",
-      "Felix Musmann",
-      "Ariane Sasso",
-      "Justin Albert",
-      "Bert Arnrich",
-    ]
-  link: "https://github.com/hpi-dhc/jointly"
-  date-accepted: 2021-12-29
-  highlight:
-  docs-url: "https://jointly.readthedocs.io"
-  citation-link: "https://doi.org/10.5281/zenodo.5833858"
-"""
+        Parameters
+        ----------
+        issue_body_list : list
+            List containing each line in the first comment of the issue
+        """
+        # Find the starting index of the section we're interested in
+        start_index = None
+        for i in range(len(issue_body_list)):
+            if issue_body_list[i][0].startswith("- Please indicate which"):
+                start_index = i
+                break
+
+        if start_index is None:
+            # If we couldn't find the starting index, return an empty list
+            return []
+
+        # Iterate through the lines starting at the starting index and grab the relevant text
+        cat_matches = ["[x]", "[X]"]
+        categories = []
+        for i in range(start_index + 1, len(issue_body_list)):  # 30):
```

Comment:
I suspect there is a more efficient way to implement this but i'm not sure what that should be.